h2o-core/src/main/java/water/rapids/ast/prims/reducers/AstTopN.java

```diff
@@ -50,65 +49,160 @@ public ValFrame apply(Env env, Env.StackHelp stk, AstRoot[] asts) { // implement
 				int colIndex = (int) stk.track(asts[2].exec(env)).getNum();     // column index of interest
 				double nPercent = stk.track(asts[3].exec(env)).getNum();        //  top or bottom percentage of row to return
 				int getBottomN = (int) stk.track(asts[4].exec(env)).getNum();   // 0, return top, 1 return bottom percentage
```

Comment:
Maybe the direction could be given by -1/1. Do you think that would make things clearer?